### ISSUE TITLE
Multi-Language Support

### ISSUE THREAD

**Issue by Wild1145** [DarthCraft/DarthCraft#22]
I think it would be cool for the ability to support multiple languages within the Darthcraft plugin.

**Comment by Wild1145** [comment]
Yes... I know it looks bad... Its been pushed back since the 2.5 planned release... Sorry...

**Comment by DUFCLiam** [comment]
@Wild1145 Please explain more. Please.

**Comment by Wild1145** [comment]
@DUFCLiam As it says on the tin, the plugin will support multiple languages.

**Comment by DUFCLiam** [comment]
Ok, so  they decide the language with a command like /language <set | change> <language> and set that in there user data? @Wild1145

**Comment by Wild1145** [comment]
Yes, something like that.
